test/Interop/Cxx/reference/reference.swift

```diff
@@ -0,0 +1,94 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-clang -c %S/Inputs/reference.cpp -I %S/Inputs -o %t/reference.o -std=c++17
+// RUN: %target-build-swift %s -I %S/Inputs -o %t/reference %t/reference.o -Xfrontend -enable-cxx-interop -Xcc -std=c++17
+// RUN: %target-codesign %t/reference
+// RUN: %target-run %t/reference
+//
+// REQUIRES: executable_test
+
+import Reference
+import StdlibUnittest
+
+var ReferenceTestSuite = TestSuite("Reference")
+
+ReferenceTestSuite.test("write-reference-in-cxx") {
```

Comment:
I'm not following the name of the test. Since we're testing the `getStaticIntRef()` function, I'd name it `read-lvalue-reference`, and for `getConstStaticIntRef()` -- `read-rvalue-reference`, WDYT?

Reply:
Done.